### PR TITLE
doc: add doc for AdGuard Home

### PR DESCRIPTION
**Pull request by Darkham42**
## Description

Add information to use AdGuard Home services.

## Type of change

- [x] Add doc

## Checklist:

- [x] I've read & comply with the [contributing guidelines](https://github.com/bastienwirtz/homer/blob/main/CONTRIBUTING.md)
- [x] I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers. 
- [x] I have made corresponding changes to the documentation (README.md).
- [x] I've checked my modifications for any breaking changes, especially in the `config.yml` file

### CODE REVIEW

File: docs/customservices.md [hunk 1]
```diff
@@ -113,3 +113,24 @@ For Prometheus you need to set the type to Prometheus and provide a url.
   url: "http://192.168.0.151/"
   # subtitle: "Monitor data server"
 ```
+
+## AdGuard Home
+For AdGuard Home you need to set the type to AdGuard, if you have somes issues as 403 responses on requests you need to provide authentification in headers for locations needed as below.
+In `config.yml`
```
Comment (bastienwirtz) [inline docs/customservices.md:119]: you can remove this line, it's the same for all services.

**Review by bastienwirtz — CHANGES_REQUESTED** [review]
Hi @Darkham42 ! 
Thanks for helping improving the doc !

File: docs/customservices.md [hunk 2]
```diff
@@ -113,3 +113,24 @@ For Prometheus you need to set the type to Prometheus and provide a url.
...
+In your conf files for your AdGuard Home instance
+```
+location /control/stats {
+  proxy_set_header Authorization "Basic [admin:password in Base64]";
+}
+
+location /control/status {
+  proxy_set_header Authorization "Basic [admin:password in Base64]";
+}
+```
```
[the remark this answers is not in the document]
Comment (Darkham42) [reply docs/customservices.md]: Ok, I've remove it.

File: docs/customservices.md (hunk 1 above)
Comment (Darkham42) [reply docs/customservices.md]: That's right. Done 😉

**Review by bastienwirtz — APPROVED** [review]

**Pull request merged**